11028 - Fix warlus operator behavior when called by a function (#11041)
pytest-dev/pytest: changelog/11028.bugfix.rst

> Fix bug related to the walrus operator in asserts. Now a variable that is assigned with the walrus operator can be used later in a function call.

Reviewer: Small suggestion, I think this is more straightforward:

```suggestion
Fixed bug in assertion rewriting where a variable assigned with the walrus operator could not be used later in a function call.
```
